operations/list.go

```diff
@@ -257,7 +264,42 @@ func listParameters(ctx context.Context, confPath, project, filename string) err
 	return nil
 }
 
-func listAliases(ctx context.Context, confPath, project, filename string) error {
+func listTriggerAliases(ctx context.Context, confPath, project, filename string) error {
+	conf, err := NewClientSettings(confPath)
+	if err != nil {
+		return errors.Wrap(err, "problem loading configuration")
+	}
+	comm := conf.setupRestCommunicator(ctx)
+	defer comm.Close()
+
+	var aliases []string
+
+	if project != "" {
+		aliases, err = comm.ListPatchTriggerAliases(ctx, project)
+		if err != nil {
+			return err
+		}
+	} else if filename != "" {
+		project, err := loadLocalConfig(filename)
+		if err != nil {
+			return err
+		}
+		aliases, err = comm.ListPatchTriggerAliases(ctx, project.Identifier)
```

Comment:
I don't understand how this works: `project.Identifier` is going to be the empty string since `loadLocalConfig` [provides the empty string as the identifier](https://github.com/evergreen-ci/evergreen/blob/020f63aae028a9d0ac0a19441bdad34a96db9cf4/operations/list.go#L356).

Reply:
It's probably a bug that listPatchAliases allows this also

Reply:
I removed it from both functions